README.md

```diff
@@ -117,6 +117,19 @@ Kubernetes Release | DigitalOcean CSI Driver Version
 
 The [DigitalOcean Kubernetes](https://www.digitalocean.com/products/kubernetes/) product comes with the CSI driver pre-installed and no further steps are required.
 
+---
+**Driver modes:**
+
+By default, driver runs in both controller and node mode. It would create disk Volumes on DigitalOcean infrastructure and mount them on the required node.
+
+Driver can be run in **controller only mode** outside DigitalOcean droplets.
+To use this mode `--region` flag (valid DigitalOcean region slug) must be provided together with `--token` flag (DigitalOcean API token).
+
+Alternatively driver can be run in **node only mode** on DigitalOcean droplets. Driver would only handle node related requests like mount volume.
```

Comment:
```suggestion
When running in **node mode only** on DigitalOcean droplets, the `token` flag is not needed (and consequently, no API token secret needs to be deployed). Additionally, in this case the `--region` and `--token` flags must not be specified together.
```

Reply:
Does this suggestion still need to be applied?

Reply:
I think yes, as it is possible to run driver in this mode if someone else would find a use case for it as we did.
Added additional sentence to be more clear how it's ran in this mode.